components/intake.py

```diff
@@ -0,0 +1,24 @@
+from magicbot import tunable
+import wpilib
+
+
+class Intake:
+    shooter_motor: wpilib.Talon
+    shoot_speed = tunable(1.0)  # speed is tunable via NetworkTables
+
+    def __init__(self):
+        self.deployed = False
+
+    def deploy(self):
+        self.deployed = True
+
+    def is_ready(self):
```

Comment:
add magicbot feedback decorator

Reply:
I think, generally for intake there can be three feedback methods
1. is_note_present
2. is_fully_retracted
3. is_fully_deployed